qcore/srf.py

```diff
@@ -9,21 +9,21 @@
 https://scec.usc.edu/scecpedia/Standard_Rupture_Format
 """
 
-from math import ceil, cos, floor, radians, sqrt
 import os
-from subprocess import Popen, PIPE
-
 import numpy as np
 
-from qcore import geo
-from qcore.config import qconfig
+from math import ceil, cos, floor, radians, sqrt, sin, degrees, atan
+from subprocess import Popen, PIPE
```

Comment:
again, the built in python modules (os, math, subprocess) have been put in separate blocks when they belong together.